### ISSUE TITLE
List requirements in README

### ISSUE THREAD

**Issue by soypat** [twharmon/goui#1]
I'm getting the following error on running a new application:

```
devserver.Server.Run: build.runDebug: exec: "wasm-opt": executable file not found in $PATH:
```

**Comment by twharmon** [comment]
Yes, more needs to be in the readme. You should have a [note](https://github.com/goui-org/gouix/blob/main/files/goui.yml#L5) on this in your goui.yml.

Change `wasm_opt` to `false` in goui.yml.

**Comment by twharmon** [comment]
I just released a newer version of gouix to make wasm-opt disabled by default.

**Comment by twharmon** [comment]
@soypat Does that resolve your issue?

**Comment by soypat** [comment]
New error message, probably need to make something in tinygo?
```
generating static assets...
compiling src...

Error: devserver.Server.Run: build.runDebug: exit status 1: error: could not find wasi-libc, perhaps you need to run `make wasi-libc`?
```

**Comment by twharmon** [comment]
I haven't seen that before. What OS are you on? Do you have tinygo installed? Run `tinygo version`.

**Comment by soypat** [comment]
Yes, tinygo is installed but I probably need to run one of the make commands to fulfill some wasm dependency I'm missing (I built tinygo from source)
